source/features/clean-pinned-issues.css

```diff
@@ -3,6 +3,15 @@
 	display: none !important; /* Hide title */
 }
 
+.rgh-clean-pinned-issues .js-pinned-issues-reorder-list {
+	gap: 16px;
+}
+.rgh-clean-pinned-issues .pinned-issue-item {
+	width: auto !important;
+	flex: 1;
+	margin-inline: 0 !important;
+}
```

Comment:
This fixes the default "3-column" layout, making it fit to 1 or 2 issues.

The single-column on the phone is unaffected:

<img width="483" alt="Screenshot" src="https://github.com/refined-github/refined-github/assets/1402241/f599ab12-b965-4d18-9384-27e11326d1cc">